storm-server/src/test/java/org/apache/storm/scheduler/resource/strategies/scheduling/TestConstraintSolverStrategy.java

```diff
@@ -551,7 +550,7 @@ public void testIntegrationWithRAS() {
     public void testZeroExecutorScheduling() {
         ConstraintSolverStrategy cs = new ConstraintSolverStrategy();
         cs.prepare(new HashMap<>());
-        Map<String, Object> topoConf = new HashMap<>();
+        Map<String, Object> topoConf = Utils.readDefaultConfig();
```

Comment:
Why are we change this in this PR?

Reply:
This test for ConstraintSolverStrategy is different internally.

Reply:
I noticed this was changed in the acker distribution PR https://github.com/apache/storm/pull/3346. I would remove this from this PR since it is irrelevant. It will be easier to understand the change when we look back the history in the future. Thanks

Reply:
Reverted